nucleus/security/core/src/main/java/com/sun/enterprise/security/admin/cli/SecureAdminConfigMonitor.java

```diff
@@ -47,15 +47,14 @@ public class SecureAdminConfigMonitor implements ConfigListener {
 
     private static final String restartRequiredMsg = Strings.get("secure.admin.change.requires.restart");
 
-    /*
-     * Must inject Domain to get notifications of SecureAdmin changes.  We
-     * cannot inject SecureAdmin itself because it might be null and that
+    /**
```

Comment:
Why promote it to javadoc?

Reply:
To make the popup visible in Eclipse. Otherwise I would use `//` instead of the "block comment".
Another question is if the content is still true...?